### ISSUE TITLE
Fill `[workspace.package]` and `[workspace.dependencies]`

### ISSUE THREAD

**Issue by StackOverflowExcept1on** [ZcashFoundation/frost#773]
https://github.com/ZcashFoundation/frost/blob/ca33dad959e55bc66d5feac53a1275faae79377b/frost-core/Cargo.toml#L1-L18
Most crate metadata can be inherited from `[workspace.package]`, see examples: [workspace](https://github.com/StackOverflowExcept1on/roast/blob/master/Cargo.toml) and [package](https://github.com/StackOverflowExcept1on/roast/blob/master/roast-core/Cargo.toml).

**Comment by conradoplg** [comment]
Good idea, thanks! When we started the project there wasn't support for this feature yet. We'll put this in our backlog.
